cli.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+const commander = require('commander');
+const fs = require('fs');
+
+const {fixDuplicates, listDuplicates} = require('./index');
+const version = require('./package.json').version;
+
+commander
+    .version(version)
+    .usage('[options] <yarn.lock path>')
+    .option('-s, --strategy <strategy>', 'Deduplication strategy. Valid values: fewer, highest. Default is "highest"', /^(fewer|highest)$/i, 'highest')
+    .option('-l, --list', 'Do not change yarn.lock, just output the diagnosis')
+    .option('--packages <packages>', 'A comma separated list of packages to dedupe. Defauls to all packages.', val => val.split(',').map(v => v.trim()))
+    .option('--print', 'Instead of saving the deduped yarn.lock, print the result in stdout')
```

Comment:
Super nipick, but the two extra options that commander adds, `--version` and `--help` start their sentences with lower case and look out of place. Might be worth making them all lowercase?

![image](https://user-images.githubusercontent.com/18694878/47765873-2b4da180-dd20-11e8-8953-d74712cb0244.png)